docs/python_armada_client.md

```diff
@@ -187,7 +187,7 @@ Uses the CreateQueue RPC to create a queue.
 
 * **Return type**
 
-    *Empty*
+    google.protobuf.empty_pb2.Empty
```

Comment:
Yeah that is very true, but I still think this gives more detail that we ideally would want to show rather than nothing? Otherwise I cannot think of a good way to do this other than writing custom code in `conf.py`.

If that is an acceptable option, I think that is the way to go. We could avoid using an old sphinx version and just do it manually.

Reply:
I mean, `conf.py` is our code? If you can make it work make it work.

Reply:
Yeah okay. I more mean by the fact it would be more code we would need to maintain - I don't know if this is nessisaraly what we'd want to do long-term, but I will definitely give it a go.

Reply:
I mean, you know how miserable the code will end up being. I think you can make a reasonable judgement over whether or not ugly is too ugly :)

Reply:
Cool :)) I wil give a few different options a go and report back